packages/block-library/src/reset.scss

```diff
@@ -6,8 +6,6 @@
  */
 
 .editor-styles-wrapper {
-	padding: 8px;
```

Comment:
This is supposed to match the frontend default padding (body), we should keep it here ideally.